backend/scripts/lint.sh

```
@@ -1,6 +1,12 @@
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+cd $SCRIPT_DIR
 cd ..
+echo "Current script parent path: $(pwd)"
+
+
 echo "Running mypy check..."
 mypy $(git ls-files "*.py")
 
+echo $(pwd)
```

Shellcheck (0.9.0-1)

[warning] Quote this to prevent word splitting.

(SC2046)